### ISSUE TITLE
Filter Menu

### ISSUE THREAD

**Issue by GVFTW** [GVFTW/collection-template#6]
**On Desktop View**
Add ease in animation or grow dropdown (like our main navigation)

<img width="901" alt="screen shot 2017-03-23 at 2 24 36 pm" src="https://cloud.githubusercontent.com/assets/3649059/24263751/7a23e942-0fd4-11e7-9d7d-32638ff9c84a.png">

--------------------------

Consistent Vertical Padding (should be 20px)

<img width="903" alt="screen shot 2017-03-23 at 2 28 17 pm" src="https://cloud.githubusercontent.com/assets/3649059/24263959/253d11f0-0fd5-11e7-936b-f880a296c708.png">

---------------------------

Checkboxes not aligned with text

<img width="73" alt="screen shot 2017-03-23 at 2 26 40 pm" src="https://cloud.githubusercontent.com/assets/3649059/24263868/da7fa7b8-0fd4-11e7-8b00-83c24483ad4a.png">

**Comment by GVFTW** [comment]
Checkboxes not aligned with text

<img width="73" alt="screen shot 2017-03-23 at 2 26 40 pm" src="https://cloud.githubusercontent.com/assets/3649059/24263868/da7fa7b8-0fd4-11e7-8b00-83c24483ad4a.png">

**Comment by Aves3888** [comment]
When a color filter is selected- there should be something showing that you clicked it - a halo or check.

**Comment by GVFTW** [comment]
@Aves3888 starting a new thread about this

**Comment by JenSparks** [comment]
@GVFTW Updated!

*Filter menu animation has been updated.

*Vertical menu padding has been updated. Now looks like this:
https://www.dropbox.com/s/c4iyasr1f45ult6/Screenshot%202017-03-30%2007.40.04.png?dl=0

*Checkboxes aligned with text.

Should look like this now:
https://www.dropbox.com/s/gnvkce373g0puqn/Screenshot%202017-03-30%2007.36.24.png?dl=0

Browsers can show form controls slightly differently (Mac v. PC v. Chrome v. Edge) which can be a pain. We'd have more control over formatting these checkboxes if we abandoned the browser default look and used a better defined style. If you provide comps or would like us to proceed in doing so, let us know.

*Filter menu colors have been updated. We made the selected state a little more obvious but can adjust if comps are provided.
